google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -353,4 +357,15 @@ protected SpannerRpc getGapicSpannerRpc() {
   public Builder toBuilder() {
     return new Builder(this);
   }
+
+  public String getEndpoint() {
```

Comment:
I assume these are copied from GrpcSpannerRpc? I think it's worth commenting.

Reply:
They are copied from here in SpannerOption: https://github.com/GoogleCloudPlatform/google-cloud-java/pull/3258/files/c39dc43a810ce92a14f9d9eddbf7803984376b61#diff-e49ba3f14ac63437c9157185d464c2a9R277

Reply:
Ah OK

Reply:
Done.